backend/apps/chat/consumers.py

```
@@ -109,10 +162,17 @@ def get_last_50_messages(self):
         qs = Message.objects.filter(room_id=self.room_id).order_by("-created_at")[:50]
         return [
             {
-                "id": m.id,
-                "parent_id": m.parent_id,
+
+                "id": getattr(m, "id", None),
+                "parent_id": getattr(m, "parent_id", None),
+                "username": getattr(m.user, "username", ""),
+                "user_id": getattr(m.user, "pk", getattr(m.user, "id", None)),
+
+                "id": m.id, # type: ignore
```

Ruff (0.15.21)

[error] Dictionary key literal `"id"` repeated

(F601)